examples/chat/src/app.tsx

```diff
@@ -34,3 +38,13 @@ export function App() {
     </AppContainer>
   );
 }
+
+createRoot(document.getElementById("root")!).render(
+  <ThemeProvider>
+    <StrictMode>
+      <JazzAndAuth>
+        <App />
+      </JazzAndAuth>
+    </StrictMode>
+  </ThemeProvider>,
+);
```

Comment:
Had to move this from main because Vite was resolving `app.tsx` as main entry.

Before this was working because main was imported here